web/client/epics/wfsquery.js

```diff
@@ -122,8 +123,10 @@ const getWFSFilterData = (filterObj) => {
 
 const getWFSFeature = (searchUrl, filterObj) => {
     const data = getWFSFilterData(filterObj);
+    const urlParsedObj = Url.parse(searchUrl);
+    const parsedUrl = urlParsedObj.protocol + '//' + urlParsedObj.host + urlParsedObj.pathname;
```

Comment:
You should mix the query string parameters (service, outputFormat) with the ones coming from the original URL. So you can keep additional parameters in the URL.